src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -105,6 +110,40 @@ export const BulkEditListFilters = ({
   const { fileUpload, isLoading } = useUpload();
   const { bulkOperationStart } = useBulkOperationStart();
 
+  const { recordTypes } = useRecordTypes();
+
+  const entityTypeDataSource = async () => {
```

Comment:
you shouldn't use ky inside components
1. move all these functions related to query to a hooks
2. it will help to increase coverage